airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/streams.py

```diff
@@ -997,6 +997,7 @@ class SellerAnalyticsSalesAndTrafficReports(IncrementalAnalyticsStream):
     result_key = "salesAndTrafficByAsin"
     cursor_field = "queryEndDate"
     fixed_period_in_days = 1
+    availability_sla_days = 2
```

Comment:
I don't see the information about SLA availability to this report as others one. See example: https://developer-docs.amazon.com/sp-api/docs/report-type-values#vendor-retail-analytics-reports
<img width="372" alt="image" src="https://user-images.githubusercontent.com/5154322/213131687-002deac9-d8a0-45db-a59d-57dcc40f1083.png">

You have any link to Amazon docs?

Reply:
I read the discussion in #18958 looks there isn't anything in Amazon docs and it is more a practical issue encountered by users. Do you mind adding a comment about it in the code.